hw1/02-palindrome.js

```diff
@@ -1,3 +1,36 @@
-const elem = document.querySelector('input');
+document.addEventListener('DOMContentLoaded', () => {
+    console.log('DOMContentLoaded event fired'); // Debugging
 
-elem.addEventListener('input', handleInput);
+    const numberInput = document.getElementById('numberInput');
+    const checkPalindromeBtn = document.getElementById('checkPalindromeBtn');
+    const resultParagraph = document.getElementById('result');
+
+    checkPalindromeBtn.addEventListener('click', () => {
```

Comment:
Similarly, consider removing the button since you can use the number input as the event to be listened for. To help me understand how to use this, I read through the docs on the number input type (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number). 

I think the example code for this Homework was really easy to use, just have to add your handleInput function 😄